src/main/java/com/bigludo/ClientHandler.java

```diff
@@ -26,10 +37,27 @@ public void sendMessage(String message) {
     @Override
     public void recieveMessage(String message) {
         System.out.println(ID + ", " + message);
-        log.debug("Receiving message" + message);
+        log.debug("Receiving message: " + "[" + message + "]");
     }
     @Override
     public String getId() {
         return ID;
     }
+    @Override
+    public void run() {
+        try {
+            while (true) {
+                String message = br.readLine();
```

Comment:
Jag skille flytta in hela läsningen av meddelandet, rad 50 till 55 i metoden receiveMessage.
Justera metoden, ta bort message-argument in.

Reply:
har flyttat om lite och lagt in raderna 50-55 i recieve message metoden samt tagit bort argumentet så att metoden nu bara kallas med recieveMessage();

Men strular lite då den nu konstant printar "client disconnected" om en client kopplar bort, är det meningen att jag ska lägga in en boolean som kollar om jag är connected eller missar jag något?

Reply:
Precis, men det har vi en annan aktivitet som vi fixar det i...

Reply:
Dra in det du har får jag se förändringarna